database_schema.md

```diff
@@ -136,6 +136,76 @@ seat_id: {
 }
 ```
 
+### Minutes Item
+```
+minutes_item: {
+    minutes_item_file_id: str
+    minutes_item_id: str
```

Comment:
Really nit picky, can you swap the order of these two.

Reply:
Done

Reply:
I believe in the old db schema, there was an `minutes_item_file` table/collection. @JacksonMaxfield wants to turn it into `event_minutes_item_file` so that the files are linked to a specific event.  We can add a field `files` to `event_minutes_item`:
```
[
  {
     event_minutes_item_file_id: str //can probably just shortened the field name to just id
     name: str
     uri: str
  }
]
```

Reply:
Sounds good, I didn't know that was a desired change so I'll make that change to `event_minutes_item`.